pkg/cluster/spec/spec.go

```diff
@@ -210,6 +210,32 @@ func (s *Specification) BaseTopo() *BaseTopo {
 	}
 }
 
+// LocationLabels returns replication.location-labels in PD config
+func (s *Specification) LocationLabels() []string {
+	lbs := []string{}
+
+	if pdReplica := s.ServerConfigs.PD["replication"]; pdReplica != nil {
```

Comment:
It's better to `merge(s.serverconfigs, pd)`. We allow users to config the `replication.location-labels` at the instance `config`. e.g:
```yaml
pd_servers:
- host: 1.2.3.4
  config:
    replication.location-labels: ...
- host: 1.2.3.5
  config:
    replication.location-labels: ...
````

Reply:
Have forbiden this case